python/practices/practice-3/src/path1/letter_validation.py

```diff
@@ -0,0 +1,40 @@
+"""
+Enter a word and a string of letters, test the word using only the
+letters in the string
+"""
+
+
+import os
+
+
+def uses_only(word, allowed_letter):
```

Comment:
I see you updated the parameter names but did not update the docstring.

Reply:
Done